lib/active_merchant/billing/gateways/nuvei.rb

```diff
@@ -123,6 +125,18 @@ def set_initiator_type(post, payment, options)
         post[:isRebilling] = stored_credentials_mode
       end
 
+      def add_account_funding_transaction(post, payment = {}, options = {})
```

Comment:
Since we are executing the AFT via the `authorize` transaction, we should always have a payment method, so I don't think you need to give it a default value.